### PR TITLE
Mention no cap on CPU minutes a month

### PR DESCRIPTION
**Pull request by fkorotkov**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ for public repositories for free:
 * 8.0 CPUs for FreeBSD VMs.
 * 2.0 CPUs macOS VM (1 VM with hyper-threading).
 
+**Note:** there are no limits on how many minutes a month you'll use! But please don't try to mine crypto coins! Most of the options are blocked via firewall rules and there are other anti-fraud mechanisms in place. Be a good citizen on OSS community!
```
Comment (RDIL) [inline docs/faq.md:35]:
```suggestion
!!! note "Monthly Minute Limit"
    There are no limits on how many minutes a month you can use! Please keep in mind that mining cryptocurrency is against our Terms of Service, and will most likely be blocked by firewall rules and other anti-fraud mechanisms. Be a good citizen in the OSS community!
```

**Review by RDIL — CHANGES_REQUESTED** [review]
Much better way of phrasing this

**Comment by fkorotkov** [comment]
@RDIL thank you!

File: docs/faq.md [hunk 2]
```diff
@@ -32,6 +32,9 @@ for public repositories for free:
 * 8.0 CPUs for FreeBSD VMs.
 * 2.0 CPUs macOS VM (1 VM with hyper-threading).
 
+!!! note "Monthly Minute Limit"
```
Comment (fkorotkov) [inline docs/faq.md:35]:
```suggestion
!!! note "No Monthly Minute Limit*"
```

Comment (RDIL) [reply docs/faq.md:35]: Yup, that works also

**Pull request merged**